modules/ROOT/pages/apikit-odatav2-extension-reference.adoc

```diff
@@ -0,0 +1,41 @@
+= APIkit for OData v2 Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+This reference contains dependency, namespace, and schema information.
+
+== Dependency Information
+
+The APIkit for OData v2 extension must be declared with the APIkit for REST module dependency:
+
+----
+<dependency>
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-module-apikit-odata</artifactId>
+    <version>2.2.0</version> //<1>
+    <classifier>mule-plugin</classifier>
+</dependency>
+<dependency>
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-apikit-module</artifactId>
+    <version>1.8.3</version> //<1>
+    <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+[calloutlist]
+.. The tooling determines the appropriate version of each dependency when scaffolding to guarantee compatibility. Manually setting dependency versions can produce issues. See the https://docs.mulesoft.com/release-notes/apikit/apikit-release-notes[APIkit release notes] before upgrading each component to the latest version.
+
+NOTE: The Anypoint Studio plugin must be updated to enable the tooling to determine the appropriate dependency combination.
```

Comment:
Suggestion (more direct)

Update the Anypoint Studio plugin to enable...

Reply:
OK! Thanks!